e2e-tests/tests/pgvector-extension/02-verify-pgvector-extension-is-enabled.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+  - script: |-
+      set -o errexit
+      set -o xtrace
+
+      source ../../functions
+
+      result=$(run_psql_local \
+        "SELECT * FROM pg_extension WHERE extname = '\''vector'\'';" \
+        "postgres:$(get_psql_user_pass pgvector-extension-pguser-postgres)@$(get_psql_user_host pgvector-extension-pguser-postgres)/postgres")
+
+      if [[ "$result" = *"vector"* ]]; then
+        echo "pgvector extension is enabled"
+      else
+        echo "pgvector extension is not enabled"
+        exit 1
+      fi
```

Comment:
I think we can keep a single if clause for the case where we want to `exit 1` the enabled case can be the default echo.

Reply:
done